packages/metal-dom/src/keyboardEvents.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import { object } from 'metal';
+
+export const KEYMAP = {
+  'BACKSPACE': 8,
```

Comment:
I have a small note on this. if instead of using an object we exported keymaps one by one we could have rollup automatically dead-code-eliminate all the stuff we don‘t use. I’ts just a few bytes but it’s still something.